<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Props Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="4">
        <widget class="QListWidget" name="ui_list_widget"/>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QComboBox" name="ui_sort_cbox">
         <item>
          <property name="text">
           <string>Date</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Name</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Size</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="ui_delete_btn">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="ui_open_btn">
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="ui_import_btn">
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" rowspan="3" colspan="2">
        <widget class="QListWidget" name="ui_info_widget"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>447</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="ui_set_dir_action"/>
   </widget>
   <widget class="QMenu" name="menuCreate">
    <property name="title">
     <string>Create</string>
    </property>
    <addaction name="ui_create_scene_action"/>
    <addaction name="ui_create_sel_action"/>
   </widget>
   <addaction name="menuCreate"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ui_set_dir_action">
   <property name="text">
    <string>Set Directory</string>
   </property>
  </action>
  <action name="ui_create_scene_action">
   <property name="text">
    <string>Create from Scene</string>
   </property>
  </action>
  <action name="ui_create_sel_action">
   <property name="text">
    <string>Create from Selection</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
